src/api/lib/tasks/flipper.rake

```diff
@@ -0,0 +1,10 @@
+namespace :flipper do
+  desc 'Enable feature toggles from ENABLED_FEATURE_TOGGLES for their group'
+  task enable_features_for_group: :environment do
+    ENABLED_FEATURE_TOGGLES.each do |feature_toggle|
+      feature_toggle_name = feature_toggle[:name]
+      # Enable the feature toggle for group with the same name
+      Flipper.enable(feature_toggle_name, feature_toggle_name)
```

Comment:
In this [old line](https://github.com/openSUSE/open-build-service/pull/12297/files#diff-a404ed7c6386193dcf5ccdb7554b6b52bd3c373205f04cc810e8ee423a19f2d9L61), we ensured we disabled the feature flag when it was fully enabled (enabled for every single user).

With your refactoring, we are not doing that anymore for that corner case. Is it no longer needed?

Reply:
alright